docs/usage.md

```diff
@@ -279,49 +269,25 @@ PBMC_10K_CMO,PBMC_10K_CMO_PBMCs_human_2,,CMO302,PBMCs_human_2
 4PLEX_HUMAN,Pancreas_BC4,BC004,,Healthy pancreas dissociated using the Miltenyi FFPE Tissue Dissociation Kit
 ```
 
-TODO: Complete the ones below. Do not have a real understanding.
+The `sample` column must match the corresponding entry in the main samplesheet.
 
-The set of CMO reference sequences with `--gex_cmo_set`.
+#### Additional reference data
 
-The frna probe reference sequences with `--gex_frna_probe_set`. Example:
+- Cellranger multi needs a reference for **GEX and VDJ analysis**. They are calculated on the fly given the reference
+  files (`--fasta` and `--gtf`) provided, but users can also provide their own with: `--cellranger_index`
+  and `--cellranger_vdj_index`, for GEX and VDJ, respectively.
 
-```csv
-#probe_set_file_format=2.0
-#panel_name=Chromium Human Transcriptome Probe Set v1.0.1
-#panel_type=predesigned
-#reference_genome=gex_reference
-#reference_version=2020-A
-gene_id,probe_seq,probe_id,included,region
-ENSG00000000003,GGTGACACCACAACAATGCAACGTATTTTGGATCTTGTCTACTGCATGGC,ENSG00000000003|TSPAN6|8eab823,TRUE,spliced
-ENSG00000000003,TCTGCATCTCTCTGTGGAGTACAATCTTCAAGTTTACAGCAACTCTTAGG,ENSG00000000003|TSPAN6|9d7fe51,TRUE,unspliced
-ENSG00000000003,AAAGCTGTTCTTAATCTCATGTCTGAAAACAAATCCTACGATGGCAGCGA,ENSG00000000003|TSPAN6|d2b5833,TRUE,spliced
-[...]
-```
+  > When running cellranger multi, without any VDJ data, users can also skip VDJ automated ref building with: `--skip_cellrangermulti_vdjref`.
 
-**Feature barcode reference for antibody capture**
-
-File containing feature barcodes used for AB analysis (`--fb_reference`). Example:
-
-```csv
-id,name,read,pattern,sequence,feature_type
-CD3,CD3,R2,^NNNNNNNNNN(BC)NNNNNNNNN,CTCATTGTAACTCCT,Antibody Capture
-CD4,CD4,R2,^NNNNNNNNNN(BC)NNNNNNNNN,TGTTCCCGCTCAACT,Antibody Capture
-CD8,CD8,R2,^NNNNNNNNNN(BC)NNNNNNNNN,GCGCAACTTGATGAT,Antibody Capture
-CD11c,CD11c,R2,^NNNNNNNNNN(BC)NNNNNNNNN,TACGCCTATAACTTG,Antibody Capture
-CD14,CD14,R2,^NNNNNNNNNN(BC)NNNNNNNNN,TCTCAGACCTCCGTA,Antibody Capture
-CD16,CD16,R2,^NNNNNNNNNN(BC)NNNNNNNNN,AAGTTCACTCTTTGC,Antibody Capture
-CD19,CD19,R2,^NNNNNNNNNN(BC)NNNNNNNNN,CTGGGCAATTACTCG,Antibody Capture
-CD56,CD56,R2,^NNNNNNNNNN(BC)NNNNNNNNN,TCCTTTCCTGATAGG,Antibody Capture
-CD45,CD45,R2,^NNNNNNNNNN(BC)NNNNNNNNN,TCCCTTGCGATTTAC,Antibody Capture
-IgG1,IgG1_control_TotalSeqC,R2,^NNNNNNNNNN(BC)NNNNNNNNN,GCCGGACGACATTAA,Antibody Capture
-A0201_NLVPMVATV_CMV_TCR-1,NLVPMVATV_CMV_TCR-1,R2,^NNNNNNNNNN(BC)NNNNNNNNN,GGCCTCGGTCCTAGG,Antibody Capture
-```
+- When working with **FFPE data**, a prob set needs to be specified via `--gex_frna_probe_set`. This file is typically
+  [provided by 10x](https://www.10xgenomics.com/support/software/cell-ranger/downloads#probe-set-downloads).
 
-gex_target_panel = null
-vdj_inner_enrichment_primers = null
-gex_barcode_sample_assignment = null
```

Comment:
Hi @grst , will we not provide information on these?

Reply:
These options are so rarely used that I don't think they warrant a description in `usage.md`. However I will improve their documentation in the nextflow schema and add references to the respective sections from the 10x documentation there.